lib/sufia/configuration.rb

```diff
@@ -84,10 +84,14 @@ def audit_user_key
       @audit_user_key ||= 'audituser@example.com'
     end
 
-    # TODO: this is called working_path in curation_concerns
```

Comment:
Hi @jcoyne , I've pushed some changes in response to your change requests. Please let me know what you think. Thanks.